hoomd/SignalHandler.cc

```diff
@@ -7,16 +7,14 @@
 #include <signal.h>
```

Comment:
This relies on the POSIX standard for `<signal.h>` to have `sigaction`, so unlike in my comment below, I don't think this can be replaced by `<csignal>`.

Reply:
When working on this, I noticed that the man page for `signal` suggests using `sigaction` instead. I also found a reference that `signal` and `sigaction` might not interoperate which I thought might be causing the issues with Jupyter.